pkg/opensearch/response_parser.go

```diff
@@ -265,6 +271,56 @@ func processTraceSpansResponse(res *es.SearchResponse, queryRes backend.DataResp
 	queryRes.Frames = data.Frames{frame}
 	return queryRes
 }
+
+func processTraceListResponse(res *es.SearchResponse, dsUID string, dsName string, queryRes backend.DataResponse) backend.DataResponse {
+	// trace list queries are hardcoded with a fairly hardcoded response format
+	// but es.SearchResponse is deliberately not typed as in other query cases it can be much more open ended
+	rawTraces := res.Aggregations["traces"].(map[string]interface{})["buckets"].([]interface{})
+
+	// get values from raw traces response
+	traceIds := []string{}
+	traceGroups := []string{}
+	traceLatencies := []float64{}
+	traceErrorCounts := []float64{}
+	traceLastUpdated := []time.Time{}
+	for _, trace := range rawTraces {
+		t := trace.(map[string]interface{})
+
+		traceIds = append(traceIds, t["key"].(string))
+		traceGroups = append(traceGroups, t["trace_group"].(map[string]interface{})["buckets"].([]interface{})[0].(map[string]interface{})["key"].(string))
```

Comment:
Did you end up trying the `flatten` helper?

Reply:
I played around with it a bit, but I don't think it helps in this case but maybe I'm misunderstanding?